internal/remote/posters.go

```
@@ -0,0 +1,164 @@
+	request, err := http.NewRequestWithContext(ctx, http.MethodGet, posterCatalogURL, nil)
+	if err != nil {
+		return PosterCatalog{}, fmt.Errorf("%w: request", ErrUnavailable)
+	}
+	response, err := client.HTTP.Do(request)
+	if err != nil {
+		return PosterCatalog{}, fmt.Errorf("%w: request failed", ErrUnavailable)
+	}
+	if response == nil || response.Body == nil {
+		return PosterCatalog{}, fmt.Errorf("%w: missing response", ErrProtocolChanged)
+	}
+	defer response.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `response.Body.Close` is not checked

(errcheck)